Magento_Theme/styles/modules/_footer-menu.scss

```diff
@@ -0,0 +1,34 @@
+.footer-menu {
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 15px 0;
+    list-style-type: none;
+    text-align: center;
+    @include mq($screen-m) {
+        display: flex;
+        flex-flow: row nowrap;
+        justify-content: center;
+        padding: 2px 0 2px 0;
+    }
+    .item {
+        display: block;
```

Comment:
currently we can't add custom class to footer links because Magento\Framework\View\Element\Html\Link\Current doesn't allow that